api/app/models/concerns/spree/webhooks/has_webhooks.rb

```diff
@@ -43,11 +43,11 @@ def resource_serializer
       end
 
       def updating_only_timestamps?
-        (saved_changes.keys - %w[created_at updated_at deleted_at]).empty?
+        (saved_changes.keys - %w[created_at updated_at destroyed_at]).empty?
```

Comment:
@aleksandarpetrushev we don't have destroyed_at column

Reply:
🤦